### ISSUE TITLE
Outdated Version Warning Emitted on Latest Version

### ISSUE THREAD

**Issue by dvnatanael** [albumentations-team/albumentations#2020]
## Describe the bug

Importing the latest version of albumentations (1.4.19) displays:

    UserWarning: A new version of Albumentations is available: 1.4.19 (you have 0.0.19). ...

### To Reproduce

1. Environment (e.g., OS, Python version, Albumentations version, etc.)

       Python version: 3.12
       Albumentations version: 1.4.19 (according to pip)

3. Sample code that produces the bug.
    
       import albumentations

4. Any error messages or incorrect outputs.

       .../lib/python3.12/site-packages/albumentations/__init__.py:24: UserWarning: A new version of Albumentations is available: 1.4.19 (you have 0.0.19). Upgrade using: pip install -U albumentations. To disable automatic update checks, set the environment variable NO_ALBUMENTATIONS_UPDATE to 1.
         check_for_updates()

### Expected behavior

There should not be any warnings, as I am already on the latest version.

### Actual behavior

A UserWarning was emitted.

### Screenshots

N/A

### Additional context

I think this was introduced by #2013: [albumentations/\_\_init__.py#L4-L5](https://github.com/albumentations-team/albumentations/blame/526187b98bb8f66b77601e9cb32e2aa24d8a76a3/albumentations/__init__.py#L4-L5)

**Comment by simonlsp** [comment]
Got same bug here.
I think the problem is that __init__.py used the metadata of albucore (which latest version is 0.0.19) for this package

```python
try:
    _metadata = metadata("albucore")
    __version__ = _metadata["Version"]
    __author__ = _metadata["Author"]
    __maintainer__ = _metadata["Maintainer"]
```

But check_version.py compared the metadata to albumentations (which latest version is 1.4.19)

```python
def fetch_version_info() -> str:
    opener = urllib.request.build_opener(urllib.request.HTTPHandler(), urllib.request.HTTPSHandler())
    url = "https://pypi.org/pypi/albumentations/json"
    try:
        with opener.open(url, timeout=2) as response:
            if response.status == SUCCESS_HTML_CODE:
                data = response.read()
                encoding = response.info().get_content_charset("utf-8")
                return data.decode(encoding)
```

**Comment by ternaus** [comment]
@dvnatanael Thank you fixed.

Will release hotfix version 1.4.20 of Albumentations later today to prevent people from getting unnecessary warnings.

**Comment by ternaus** [comment]
Released version 1.4.20
